Gemfile.lock

```diff
@@ -33,7 +33,7 @@ GEM
     github-pages (186)
       activesupport (= 4.2.10)
       github-pages-health-check (= 1.8.1)
-      jekyll (= 3.7.3)
+      jekyll (= 3.7.4)
```

Comment:
Do not update Gemfile.lock it's generated.  The best way is update whole github-page in Gemfile. Just change 6 line from gem 'github-pages' to gem 'github-pages', '192'.

Reply:
ok. can you make this change yourself please?